tests/editing/totals_and_components/test_totals_and_components.py

```diff
@@ -6393,7 +6719,31 @@ def test_correct_components(
             )
 
 
-def compare_list_of_floats(components: List[float], expected_components: List[float]):
+def compare_results_to_expected_results(
+    results: TotalsAndComponentsOutput, expected_result: tuple
+):
+    assert results.identifier == expected_result[0]
+    assert results.absolute_difference == expected_result[1]
+    assert results.low_percent_threshold == expected_result[2]
+    assert results.high_percent_threshold == expected_result[3]
+    assert results.final_total == expected_result[4]
+    assert results.tcc_marker == expected_result[6]
+
+    if results.tcc_marker == "T" or results.tcc_marker == "C":
+        sum_of_components = 0
+        for component in results.final_components:
+            print(f"fc - {results.final_components}")
```

Comment:
This will print all component for every component.  Maybe you wanted this for debug, but I think it can be removed.